auth-api/migrations/versions/2024_05_15_60711696cdd2_add_gov_flags_business_type_codes.py

```diff
@@ -0,0 +1,113 @@
+"""add_business_and_government_flags_and_remove_rows
+
+Revision ID: 60711696cdd2
+Revises: e2d1d6417607
+Create Date: 2024-05-15 15:04:38.643180
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '60711696cdd2'
+down_revision = 'e2d1d6417607'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('business_type_codes', sa.Column('is_government_agency', sa.Boolean(), nullable=True))
+    op.add_column('business_type_codes', sa.Column('is_business', sa.Boolean(), nullable=True))
+
+    mapping = [
+        ("BIZ", False, True),
+        ("BIZAC", False, True),
+        ("BIZEL", False, True),
+        ("BIZGA", False, True),
+        ("BIZGE", False, True),
+        ("BSPLY", False, True),
+        ("CAR", False, True),
+        ("CAREQ", False, True),
+        ("CON", False, True),
+        ("EDUC", False, True),
+        ("EXPLR", False, True),
+        ("FINBA", False, True),
+        ("FINCO", False, True),
+        ("FINCU", False, True),
+        ("FINIC", False, True),
+        ("FINLE", False, True),
+        ("FINTR", False, True),
+        ("FIRNA", False, True),
+        ("FOR", False, True),
+        ("GOVA", True, False),
+        ("GOVAA", True, False),
+        ("GOVCC", True, False),
+        ("GOVD", True, False),
+        ("GOVF", True, False),
+        ("GOVGA", True, False),
+        ("GOVL", True, False),
+        ("GOVLT", True, False),
+        ("GOVM", True, False),
+        ("GOVOP", True, False),
+        ("GOVP", True, False),
+        ("GOVR", True, False),
+        ("GOVSR", True, False),
+        ("HLTH", True, False),
+        ("INSUR", False, True),
+        ("INTRN", False, False),
+        ("LAW", False, True),
+        ("LAWNP", False, True),
+        ("LDB", False, True),
+        ("LDBC", False, False),
+        ("MAR", False, True),
+        ("OTHCC", False, True),
+        ("OTHCR", False, True),
+        ("OTHER", True, True),
+        ("OTHNE", False, True),
+        ("OTHUN", False, True),
+        ("PILOT", False, False),
+        ("REAL", False, True),
+        ("REALA", False, True),
+        ("REALB", False, True),
+        ("RESRC", False, True),
+        ("SES", True, False),
+        ("SURV", False, True),
+        ("TSC", False, True)
+    ]
+
+    for code, is_government_agency, is_business in mapping:
+        op.execute(f"UPDATE business_type_codes SET is_government_agency = '{is_government_agency}', is_business = '{is_business}' WHERE code = '{code}'")
+
+    # Delete rows that need to be removed
+    codes_to_remove = [
+        "PILOT",  # HRV
```

Comment:
HRV?

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/16507223/4628347b-ddc3-49f0-99f5-4ae4ed787bbb)

Reply:
What does it mean?

Reply:
I don't know what this code mean, HRV is the description of code "PILOT" provided by https://docs.google.com/spreadsheets/d/1hGi4VUnpl0MBdd0yiDIdZckJd3lccQ9vOlMNLtnArSE/edit#gid=0, this line need to be removed, cause it's no longer a product.

Reply:
no worries, was trying to figure out what it meant :P